hiv-benchmarking/hiv_raw_data/README.md

```diff
@@ -0,0 +1,52 @@
+### Raw Data Sources:
+
+
+#### 1. 5_min_75_confidence.sif
+- Source: https://github.com/gitter-lab/hiv1-aurkb/blob/main/Results/networks/hiv-networks.cys
+- Downloaded the 75% confidence-filtered network at 5 min as a .sif file after opening the file in Cytoscape and removed the '%' char from the filename to prevent any issues with reading file
+
+
+#### 2. correctedprize_05.csv and correctedprize_060.csv
+- todo
```

Comment:
There are some todos in this file

Reply:
correctedprize_05.csv and correctedprize_060.csv are the prize files from the private hiv repo. what's the best way to link the source for this?

Reply:
I'll provide text suggestions. It had been a while, so I forgot we have both `correctedprize_05.csv` and `prize_05.csv` in this pull request and are comparing them.